SV blog post (#4354)
Couchers-org/couchers: app/web/markdown/blog/2024/05/20/introducing-strong-verification.md

---
title: Introducing Strong Verification
slug: introducing-strong-verification
description: We are excited to announce a new verification feature called "Strong Verification". This feature increases Trust and Safety on the platform by allowing users to verify their information using a biometric passport.
date: 2024/05/20
author: Aapeli
# todo: picture of biometric passport
share_image: https://couchers.org/img/share.jpg
---

Today we're announcing a new feature called Strong Verification. You can complete Strong Verification today using your biometric passport and an NFC enabled mobile phone.

> Trust and Safety are a core tenet of usability for any couch surfing platform. At Couchers, we've spent a lot of time thinking about how we could make the platform safer and give people more control in order to use the platform according to their needs and preferences. Today we're announcing a new feature that gives users more power and peace of mind when using Couchers.org.

Reviewer: At Couchers, we've spent a lot of time thinking about how we could make the platform safer and reassuring

Author: I'm not sure what you meant here!

Reviewer: "and give people more control in order to use the platform according to their needs and preferences" is not really what we want to convey because there are a lot of people out there that would like to use it like tinder/airbnb/etc. We should focus on the safety aspect of it

> Trust and Safety are a core tenet of usability for any couch surfing platform. At Couchers, we've spent a lot of time thinking about how we could make the platform safer and give people more control in order to use the platform according to their needs and preferences. Today we're announcing a new feature that gives users more power and peace of mind when using Couchers.org.

Reviewer: Today we're announcing a new feature that improves verification and gives users peace of mind when using Couchers.org.

Your data will be securely stored on the server, with the most sensitive data encrypted and inaccessible to the server itself. You can read more about how we store and process your data [here](link).

## Is Strong Verification free? Do I have to pay for it?

**Every user will be able to complete Strong Verification for free until 1st October, 2024.**

Strong Verification costs the Couchers, Inc. 501(c)3 non-profit real money. We do not currently have an abundance of funds, and so the board will have to make a decision regarding funding for this feature. Please [consider donating](https://couchers.org/donate).

## What information is verified?

We verify your date of birth and gender^[footnote: passports do not contain information about gender, only sex. We are forced to make a simplifying assumption regarding sex and gender. We also have a mechanism for those for whom this assumption does not hold, you can find out more [here](todo link).] match the details listed on your profile.

## Why should I complete Strong Verification?

> By completing Strong Verification, other users will have the peace of mind that they are interacting with a real human. Strong Verification also stops people who were banned for violating the terms of service or community guidelines from creating new verified accounts^[footnote: Each passport can opnly be verified once, so users who are banned cannot re-verify using the same passport.]. Additionally, this data may be helpful if we ever need to collaborate with criminal investigations.

Reviewer: Additionally, this data may be helpful if we ever need to collaborate with a criminal investigation.

Finally, users with Strong Verification will show up higher in search results and have a higher chance of both receiving requests and having their host requests accepted. In the coming weeks and months, we will be rolling out further features based on Strong Verification, providing further incentive to verify.

## Is Strong Verification compulsory?

> No, we believe in giving all of our community members the ability to use the platform how they wish, as long as they follow the community guidelines and terms of service, and follow the spirit of our community. We know some members do not feel comfortable verifying their information using this mechanism, and that not all members are able to do verification, even if they would like to do so; either because they don't have a compatible passport or a compatible phone. Users who do not complete Strong Verification will still be able to use the full features of the platform, except those few features that build on top of Strong Verification. Strong Verificaiton increases visibility and trust, but those who do not wish to complete it will still be able to use Couchers to the full extent.

Reviewer: No. We know some members do not feel comfortable verifying their information using this mechanism, and that not all members are able to do verification, even if they would like to do so; either because they don't have a compatible passport or a compatible phone.

---
title: Introducing Strong Verification
slug: introducing-strong-verification
description: We are excited to announce a new verification feature called "Strong Verification". This feature increases Trust and Safety on the platform by allowing users to verify their information using a biometric passport.
date: 2024/05/20
author: Aapeli
# todo: picture of biometric passport
share_image: https://couchers.org/img/share.jpg
---

Today we're announcing a new feature called Strong Verification. You can complete Strong Verification today using your biometric passport and an NFC enabled mobile phone.

Trust and Safety are a core tenet of usability for any couch surfing platform. At Couchers, we've spent a lot of time thinking about how we could make the platform safer and reassuring. Today we're announcing a new feature that improves verification and gives users peace of mind when using Couchers.org.

> **You can complete Strong Verification today, by vising the [new verification settings page](todo link).**

Reviewer 2: ```suggestion
**You can complete Strong Verification in less than 5 minutes, by visiting the [new verification settings page](todo link).**
```

## How does it work?

> To complete Strong Verification, you need your biometric passport, an NFC capable Android or iOS device, and the Iris ID app. You can download the app from the [Apple App Store](https://apps.apple.com/us/app/iris-id/id1575142357), or the [Google Play store](https://play.google.com/store/apps/details?id=seismic.rarity).

Reviewer 2: ```suggestion
To complete Strong Verification, you need your biometric passport, an NFC capable Android or iOS device, and the Iris ID app. You can download the app from the [Apple App Store](https://apps.apple.com/us/app/iris-id/id1575142357), or the [Google Play Store](https://play.google.com/store/apps/details?id=seismic.rarity).
```

> You will first need to initiate Strong Verification in the new [verification settings page](todo link). You will be presented with a link or QR code to open in the Iris ID app. After following this link, you will be asked to scan the picture page of your passport, and then touch your NFC device to your passport while the information is downloaded from your passport. The data will be passed onto the Couchers server, encrypted, and you will receive a badge on your profile, as well as indicators for verified information.

Reviewer 2: ```suggestion
You will first need to initiate Strong Verification in the new [verification settings page](todo link). You will be presented with a link or QR code to open in the Iris ID app. After following this link, you will be asked to scan the picture page of your passport, and then touch your NFC device to your passport while the information is downloaded from your passport. The data will be securely passed onto the Couchers server, encrypted, and you will receive a badge on your profile, as well as indicators for verified information.
```

Your data will be securely stored on the server, with the most sensitive data encrypted and inaccessible to the server itself. You can read more about how we store and process your data [here](https://help.couchers.org/hc/couchersorg-help-center/articles/1716502273-technical-description-of-strong-verification).

## Is Strong Verification free? Do I have to pay for it?

> **Every user will be able to complete Strong Verification for free until 1st October, 2024.**

Reviewer 2: ```suggestion
**Every user will be able to complete Strong Verification for free until 1st October, 2025.**
```
